src/components/shared/Label/index.tsx

```diff
@@ -5,13 +5,14 @@ type LabelProps = {
   children: React.ReactNode;
   htmlFor: string;
   className?: string;
+  ariaLabel?: string;
 };
 
-const Label = ({ children, htmlFor, className }: LabelProps) => {
+const Label = ({ children, htmlFor, className, ariaLabel }: LabelProps) => {
   const classes = classnames('usa-label', className);
 
   return (
-    <label className={classes} htmlFor={htmlFor}>
+    <label className={classes} htmlFor={htmlFor} aria-label={ariaLabel}>
```

Comment:
Having an aria label is a best practice across the board, right? In previous apps, I've defaulted the `ariaLabel` to be the same as the `htmlFor` unless we passed something in explicitly. Would it be good to do something like that here so we aren't setting it to null for a lot of fields?

Reply:
My understanding of `aria-label`s are that they are only necessary when the text or label is not descriptive enough for assistive technology. The most common use case is adding `aria-label` for icons or labels/text that aren't descriptive enough.

I think `htmlFor` needs to be unique identifier tied to a HTML ID. Since `aria-label`'s aren't always unique, I'm not sure if it's a good idea to merge the two.

Having `aria-label="null"` would be bad. I need to double check and make sure that doesn't happen when `aria-label` is not passed to the component. I was under the impression that React is smart enough to omit it.

Reply:
I double checked the `Label` components without the `ariaLabel` set and it looks like React is smart enough to omit the prop when it's not defined.

![Screen Shot 2020-06-11 at 10 57 22 PM](https://user-images.githubusercontent.com/8367504/84470097-68d21a80-ac37-11ea-8c3d-eb64cb2af309.png)